inspirehep/config.py

```diff
@@ -234,3 +303,27 @@ def _(x):
 
 RECORDS_DEFAULT_STORAGE_CLASS = "S"
 """Default storage class for record files."""
+SEARCH_SOURCE_INCLUDES = {
+    "literature": [
+        "$schema",
+        "abstracts.value",
+        "arxiv_eprints.value",
+        "arxiv_eprints.categories",
+        "authors.affiliations",
```

Comment:
\+ `authors.inspire_roles`

Also could you make sure it's synced with the whole list.